msbuild/domoticz.vcxproj

```diff
@@ -618,6 +618,8 @@
     <ClInclude Include="..\main\WindCalculation.h" />
     <ClInclude Include="..\smtpclient\SMTPClient.h" />
     <ClInclude Include="..\sqlite\sqlite3.h" />
+    <ClInclude Include="..\tinyxpath\tinyxml.h" />
```

Comment:
I've added the directory (after finding the correct path specification) and the builds are now succeeding.